src/platform/linux.rs

```diff
@@ -47,7 +47,7 @@ fn time(on_ac: bool, charge_full: i32, charge_now: i32, current_now: i32) -> Dur
     }
 }
 
-/// Parse an unsigned integer out of a string, surrounded by whitespace
```

Comment:
huh, does new nom expand these into something that doesn't take documentation?? o_0

Reply:
the compiler warning/suggestion was: 'to document an item produced by a macro, the macro must produce the documentation as part of its expansion'

...so if I have understood this correctly, you can't put a docstring over any macro, because the macro itself would have to generate a docstring during its expansion. Am I completely wrong/ overlooking something obvious here or does this not have anything to do with nom? :D

Reply:
well that's weird.. okay whatever